src/Nordea/Components/Button.elm

```diff
@@ -85,8 +89,11 @@ import Nordea.Themes as Themes
 
 type Variant
     = Primary
+    | PrimarySmall
     | Secondary
+    | SecondarySmall
     | Tertiary
+    | TertiarySmall
     | Card
     | SmallCard
```

Comment:
is it worth renaming this too for consistency?

Reply:
Yeah, I can do it in a seperate PR ☺️